index.html

```diff
@@ -2437,6 +2463,9 @@ <h4>
             MUST run the following steps:
           </p>
           <ol>
+            <li>
+              <a>Assert</a>: this is running <a>in parallel</a>.
+            </li>
```

Comment:
Given your last comment and update, this algorithm does not run in parallel, only specific steps do.

```suggestion
```

Reply:
Done.